sycl/doc/extensions/experimental/sycl_ext_matrix/sycl_ext_oneapi_matrix.asciidoc

```diff
@@ -439,6 +537,22 @@ void joint_matrix_load(Group g,
     joint_matrix<Group, precision::tf32, Use, Rows, Cols, Layout> &res,
     multi_ptr<float, Space, IsDecorated> src, size_t stride);
 
+template <typename Group, size_t Rows, size_t Cols,
+          typename PropertyListT>
+void joint_matrix_load(Group g,
+    joint_matrix<Group, precision::tf32, use::accumulator,
+                 Rows, Cols, layout::dynamic> &res,
+    ext::oneapi::experimental::annotated_ptr<float, PropertyListT> src,
+    size_t stride, layout Layout);
+
+// Only available when Layout != layout::dynamic
+template <typename Group, size_t Rows, size_t Cols, use Use,
+          layout Layout, typename PropertyListT>
+void joint_matrix_load(Group g,
+    joint_matrix<Group, precision::tf32, Use, Rows, Cols, Layout> &res,
+    ext::oneapi::experimental::annotated_ptr<float, PropertyListT> src,
+    size_t stride);
+
```

Comment:
In the other PR I reviewed, I proposed a different way to handle these special TF32 overloads.  Notice that with that proposal, you wouldn't need to add these new TF32 overloads for `annotated_ptr` because you would just use the regular non-TF32 versions.

Reply:
Yes, let's solve this in  https://github.com/intel/llvm/pull/12276.
I will keep it this way for now because this is how it is done for the other load. Whichever PR gets merged first, the other one will follow.